requirements.txt

```diff
@@ -1,14 +1,13 @@
 diffimg
 matplotlib
 numpy<2
+pygmt_helper @ git+https://github.com/ucgmsim/pygmt_helper@cylc
```

Comment:
Can the changes that are needed for these to work be merged for these 2 repos with a PR? That way we can try use main, if not maybe we just write a TODO here to update these once we can get those changes in

Reply:
I don't actually think that needs to be a cylc branch. It did at some point but I am no longer sure it does.